README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/AEGEE/oms-mailer.svg?branch=master)](https://travis-ci.org/AEGEE/oms-mailer)
+[![Build Status](https://travis-ci.org/AEGEE/mailer.svg?branch=master)](https://travis-ci.org/AEGEE/mailer)
```

Comment:
Maybe update this to Circle CI as well?

Reply:
Nor sure if CircleCI provides badges, will check later.

Reply:
yeah, it does, updated it. I've changed it to mailer as I'll rename the repo before merging.